components/locale/README.md

```diff
@@ -9,3 +9,14 @@ The project is in an incubation period.
 # Authors
 
 The project is managed by a subcommittee of ICU-TC in the Unicode Consortium focused on providing solutions for client-side internationalization.
+
```

Comment:
I would either move this section into documentation/build.md or put more description here (or drop it from this PR).
Filip did a nice index.md + README.md edit in #69